common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch6/IndexSettings.scala

```diff
@@ -0,0 +1,30 @@
+package com.gu.mediaservice.lib.elasticsearch6
+
+import com.sksamuel.elastic4s.analyzers._
+
+object IndexSettings {
+
+  val hierarchyAnalyserName = "hierarchyAnalyzer"
+  // TODO rename `english_s_stemmer` as its an analyzer not a stemmer - would require a reindex.
+  val englishSStemmerAnalyzerName = "english_s_stemmer"
+
+  def analysis = {
+    val guStopWords = StopTokenFilter(name = "gu_stopwords", stopwords = Seq("_english_"))
+
+    val sStemmer = StemmerTokenFilter(name = "s_stemmer", lang = "minimal_english")
+
+    val englishPossessiveStemmer = StemmerTokenFilter(name = "english_possessive_stemmer", lang = "possessive_english")
+
+    val englishSStemmerAnalyzer = CustomAnalyzerDefinition(name = englishSStemmerAnalyzerName,
+      tokenizer = StandardTokenizer,
+      filters = Seq(LowercaseTokenFilter, AsciiFoldingTokenFilter, englishPossessiveStemmer, guStopWords, sStemmer))
```

Comment:
Completely unrelated to this PR, but we should one day swap asciiFolding for [ICU Normalizer](https://www.elastic.co/guide/en/elasticsearch/plugins/current/analysis-icu.html). Looks much more powerful.